src/debug/debuggerConfigureDriver.ts

```diff
@@ -0,0 +1,26 @@
+import { CMakeOutputConsumer } from "@cmt/diagnostics/cmake";
+import * as vscode from "vscode";
+
+export const debuggerPipeName = "\\\\.\\\\pipe\\\\cmake-debugger-pipe";
+
+export async function startConfigureDebugger(outputConsumer: CMakeOutputConsumer): Promise<void> {
+    if (outputConsumer instanceof CMakeOutputConsumer) {
```

Comment:
Is this check necessary? If so, then the type annotation on the parameter should be relaxed to capture the range of types we actually expect to be passed to this function.

Reply:
Good point. I only expect it to ever be a CMakeOutputConsumer, so I'll remove the check.

Reply:
Removing this check.